roofit/roofitcore/src/TestStatistics/LikelihoodJob.h

```diff
@@ -69,14 +76,16 @@ class LikelihoodJob : public MultiProcess::Job, public LikelihoodWrapper {
    RooArgList vars_;      // Variables
    RooArgList save_vars_; // Copy of variables
 
-   LikelihoodType likelihood_type_;
    std::size_t n_tasks_at_workers_ = 0;
 
    // warning: don't use the following values directly, use the getters instead!
    std::size_t n_event_tasks_;
    std::size_t n_component_tasks_;
    std::size_t getNEventTasks();
    std::size_t getNComponentTasks();
+
+   std::vector<ROOT::Math::KahanSum<double>> offsets_previous_;
+   std::unique_ptr<LikelihoodSerial> likelihood_serial_;
```

Comment:
```suggestion
   LikelihoodSerial likelihood_serial_;
```
Why you're using a pointer here? In the constructor, the type looks to be clear at compile time:
```c++
     likelihood_serial_(std::make_unique<LikelihoodSerial>(likelihood_, calculation_is_clean_, component_offsets_,
                                                           component_offsets_save_))
```
Then, you also don't have to define the copy constructor manually, which is very error-prone if data members are added.

Reply:
Replaced the pointer with a member.

Regarding copying: I decided to forbid it completely. It's not currently used and I don't think it should be. If the MinuitFcnGrad ever is copied, it should probably just create new Likelihood(Gradient)Wrappers. In any case, for the time being they are superfluous and as you say error prone. I basically just don't want to introduce them right now for the reason you mention: keeping them up to date will be a hassle (they are not trivial due to the shared offset object... what does copying mean? Would a user want it with a new offset or with the old one? I don't even think I can know, only the manager would know, like MinuitFcnGrad). So let's just leave them out to make sure when ever they do become necessary someone will make sure they will be fit for purpose at that time.